content/pages/about.html

```diff
@@ -0,0 +1,396 @@
+<html>
+  <head>
+    <title>About</title>
+    <meta name="template" content="page" />
+    <meta name="status" content="hidden" />
+    <meta name="slug" content="about" />
+  </head>
+  <body>
+
+{% import "partials/homepage.html" as homepage %}
+{% import "partials/about.html" as about %}
+
+<section>
+  <div class="ui horizontally very padded container">
+    <div class="ui very padded centered grid">
+      <div class="middle aligned row">
+
+        <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
+          <div class="ui large left aligned basic vertical segment">
+            <h1 class="ui huge header">
+              How documentation automation works with Read the Docs
+            </h1>
+
+            {% markdown %}
+            Automate the deployment of your technical documentation
+            by building your project with every change to your documentation source.
+            Maintain your documentation without managing extra services.
+            Building, hosting, CDN, authentication, and search are all included.
+            {% endmarkdown %}
+
+          </div>
+        </div>
+
+        <div class="six wide computer twelve wide tablet sixteen wide mobile column">
+          <div class="ui padded basic segment">
+            <div class="ui relaxed list">
+
+              {%- set header_1 = "Maintain docs as code" %}
+              {%- set icon_1 = "fa-code-branch" -%}
+              <a href="#authoring" class="ui small teal header item">
+                <i class="fad {{ icon_1 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_1 }}
+                  <span class="sub header">Use your existing development workflow to maintain your documentation</span>
+                </span>
+              </a>
+
+              {%- set header_2 = "Build automatically" %}
+              {%- set icon_2 = "fa-gears" -%}
+              <a href="#building" class="ui small header item">
+                <i class="fad {{ icon_2 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_2 }}
+                  <span class="sub header">Update your documentation whenever you push changes</span>
+                </span>
+              </a>
+
+              {%- set header_3 = "Preview your changes" %}
+              {%- set icon_3 = "fa-code-compare" -%}
+              <a href="#review" class="ui small header item">
+                <i class="fad {{ icon_3 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_3 }}
+                  <span class="sub header">Verify changes to your documentation during review</span>
+                </span>
+              </a>
+
+              {%- set header_4 = "Everything in one place" %}
+              {%- set icon_4 = "fa-box-check" -%}
```

Comment:
nit: Can we align this with the other headers?

![Screenshot_2022-08-18_11-53-45](https://user-images.githubusercontent.com/244656/185367243-f4a67ecf-d883-4d42-b5e6-4303ccb1c03d.png)

Reply:
@humitos Where are you seeing that? I can't reproduce it.

Reply:
Oh, looks like on a small screen size? It does seem like each heading is pushed in slightly to the right:
![read-the-docs-site-community--147 com readthedocs build_en_147_about_](https://user-images.githubusercontent.com/25510/185443602-8ea54e5f-0319-402e-aada-a2e78bb65b50.png)

Reply:
Aye. Likely a column width issue.

Reply:
I did a light restructure of this section, dropping redirects down to the additional features section. For now, the content on the right mentioned privacy levels, as that's another common question we get.

Long term, I think the hosting section could use a lot more information on authentication (privacy levels, sharing tokens, SSO!), but would also like to see a graphic replace the content block I've added on the right column too.

![image](https://user-images.githubusercontent.com/1140183/187614278-baefc705-ae40-4dd8-8a3f-b3994772aa49.png)

Reply:
Let's ship it!

Reply:
@agjohnson It still looks like the original issue this is talking about is there, the columns are shifted to the right each time down the page. But I still think we ship it, then fix it in another PR.